pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -167,6 +197,54 @@ func (c *podIpMetricsCollector) listIpAddresses(dir string) error {
 	return nil
 }
 
+func getIPv4(s string) (bool, string) {
+	names := strings.Split(s, "/")
+	f := names[len(names)-1]
+	ip := net.ParseIP(f)
+	if ip == nil || ip.To4() == nil {
+		return false, ""
+	}
+	return true, f
+}
+
+// After the ip is removed, it will be put into the reuseMap.
+// After the ip is reused, the reuseMap will update the ip reuse time.
+// The reuseMap will not record any ip that is being used, only keeps the reused one.
+func (c *podIpMetricsCollector) updateReuseIps(e fsnotify.Event, reuseMap map[string]*ipReuse, f string, bucketKeys []float64) {
```

Comment:
Better to rename this updateReuseIpStats

Reply:
Done.